calculator.go

```diff
@@ -1,13 +1,35 @@
 // Package calculator provides a library for simple calculations in Go.
 package calculator
 
+import "errors"
+
 // Add takes two numbers and returns the result of adding them together.
 func Add(a, b float64) float64 {
-return a + b
+	return a + b
 }
 
 // Subtract takes two numbers and returns the result of subtracting the second
 // from the first.
 func Subtract(a, b float64) float64 {
 	return b - a
 }
+
+// Multiply takes two numbers and returns the result of multiplying the two.
+func Multiply(a, b float64) float64 {
+	return a * b
+}
+
+// Divide takes two numbers and returns the result of Divide the two.
+func Divide(a, b float64) (float64, error) {
+	if b == 0 {
+		return 0, errors.New("Invalid")
```

Comment:
Good! We could do more here, couldn't we? We could say _why_ it's invalid, which might not otherwise be obvious without reading the source code:

```suggestion
		return 0, errors.New("division by zero is undefined")
```